definitions/procedures/pulp/remove.rb

```diff
@@ -163,12 +174,33 @@ def drop_migrations
     end
     # rubocop:enable Metrics/BlockLength
 
-    def delete_pulp_data(rm_cmds)
+    def delete_pulp_data
+      non_mountpoints = data_dir_removal_cmds(deletable_pulp_dirs)
+      mountpoints = pulp_data_dirs_mountpoints
       with_spinner('Deleting pulp2 data directories') do |spinner|
-        rm_cmds.each do |cmd|
-          execute!(cmd)
+        if non_mountpoints.any?
+          non_mountpoints.each do |cmd|
+            execute!(cmd)
+          end
         end
-        spinner.update('Done deleting pulp2 data directories')
+        if mountpoints.any?
+          msg_for_del_mountpoints
+        else
+          spinner.update('Done deleting pulp2 data directories')
+        end
+      end
+    end
+
+    def msg_for_del_mountpoints
+      mountpoints = pulp_data_dirs_mountpoints
+      if mountpoints.count > 1
+        puts "\nThe directories #{mountpoints.join(',')} are individual mountpoints."\
+              "\nThe satellite-maintain won't delete these directories.\n"\
```

Comment:
It's foreman-maintain, please ;)

Reply:
Now calling `ForemanMaintain.pkg_and_cmd_name` which should render the cmd as per features :)